presto-spi/src/test/java/com/facebook/presto/spi/page/TestPageCodecMarker.java

```diff
@@ -84,4 +87,19 @@ public void testSummaryString()
             assertTrue(allMarkersSummary.contains(marker.name()));
         }
     }
+
+    @Test
+    public void testComputeCRC() {
+        Slice data = utf8Slice("This is a random text");
+        byte pageCodecmarker = 4;
+        int positionCount = 12444567, uncompressedSize = 10020078;
```

Comment:
The common practice is to define each variable separately. Here, however, there doesn't seem to be any need for these variables. Inlining might be better:

```computeSerializedPageChecksum(data, pageCodecMarker, 12444567, 10020078)```

Reply:
Oh this is what you meant by inline. Fixing.